tools/DevicesApiTester/Commands/Pwm/DimLed.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Pwm;
+using System.Threading.Tasks;
+using CommandLine;
+using DeviceApiTester.Infrastructure;
+
+namespace DeviceApiTester.Commands.Pwm
+{
+    [Verb("pwm-dim-led", HelpText = "Dims an LED connected to a specified PWM pin for 3 seconds")]
+    public class DimLed : PwmCommand, ICommandVerbAsync
+    {
+        /// <summary>Executes the command asynchronously.</summary>
+        /// <returns>The command's exit code.</returns>
+        /// <remarks>
+        ///     NOTE: This test app uses the base class's <see cref="CreatePwmController"/> method to create a device.<br/>
+        ///     Real-world usage would simply create an instance of <see cref="PwmController"/>:
+        ///     <code>using (var pwm = new PwmController())</code>
+        /// </remarks>
+        public async Task<int> ExecuteAsync()
+        {
+            Console.WriteLine($"LedPin={LedPin}, DutyCycle={DutyCycle}, Frequency={Frequency}, Driver={Driver}");
+
+            using (var pwm = CreatePwmController())
+            {
+                int pwmChip = 0;
+                int pwmChannel = LedPin;
+                double pwmFreq = (double)Frequency;

```

Comment:
>(double)Frequency; [](start = 33, length = 18)

If Frequency needs to be double, maybe the property should just be double too?

Reply:
I feel like frequency should be an integer :) can't see a use for i.e. 3.7Hz...

Reply:
yes, will make it a double

Reply:
using double since the sample can be used for more than LED dimming